restful-ws/3.0/_index.md

```diff
@@ -33,19 +33,20 @@ Please reference that ballot for the official results.
 
```

Comment:
Why was this update added to this PR?  I think we have a separate PR #300 for jax-rs ballot results.

Reply:
Should be fixed, I forgot which repo I was using for this change